pom.xml

```diff
@@ -465,8 +465,10 @@
         </repository>
         <repository>
             <id>osgeo</id>
-            <name>Open Source Geospatial Foundation Repository</name>
-            <url>https://download.osgeo.org/webdav/geotools/</url>
+            <name>OSGeo Release Repository</name>
+            <url>https://repo.osgeo.org/repository/release/</url>
```

Comment:
I'm ok with this change, though it relates not originally with this issue?

Reply:
No, it doesn't. geotools has moved to another repo: https://github.com/opentripplanner/OpenTripPlanner/issues/3034